bedrock/privacy/templates/privacy/archive/index.html

```diff
@@ -62,19 +68,25 @@ <h2>Hello</h2>
   <section>
     <h2>Persona</h2>
     <ul class="mzp-u-list-styled">
-      <li><a href="{{ url('privacy.archive.persona-2017-07') }}"><time datetime="2017-07">July 2017</time></a></li>
```

Comment:
Question, non blocking: why did we drop the `time` elements? Just curious

Reply:
It’s inconsistent throughout the template, and for such a simple format it seems like superfluous markup for markup’s sake imho